monkey/monkey_island/cc/next_ui/src/lib/styling/increaseContrast.ts

```diff
@@ -0,0 +1,12 @@
+import { darken, lighten } from '@mui/system';
+
+const increaseContrast = (color, theme, times = 1) => {
+    const tonalOffset = times * theme.palette.tonalOffset;
```

Comment:
Is it possible for tonalOffset to overflow the color range?

Reply:
No, it will converge to black/white